### PR TITLE
Add another QUILT option 'regional_latlon'

### PR DESCRIPTION
**Pull request by chan-hoo**
## DESCRIPTION OF CHANGES: 

- Add another QUILT option `regional_latlon` (parameter `WRTCMP_output_grid` in config.sh).

## TESTS CONDUCTED: 

- On machines/platforms:

- [x] WCOSS2
- [ ] Hera
- [ ] Orion
- [ ] Jet

- Test cases: 

- [x] Non-DA engineering test
- [ ] DA engineering test
- [x] Other sample scripts: add the following parameter to the non-DA sample config script 
```
WRTCMP_output_grid="regional_latlon"
```

## ISSUE: 

- Fixes the issue(s) mentioned in #83

### CODE REVIEW

File: scripts/exrrfs_run_fcst.sh
Comment (MatthewPyle-NOAA) [inline scripts/exrrfs_run_fcst.sh:1]: Another change that seems a bit outside of the main emphasis of this PR ; is this just fixing an issue that was exposed along the way?

**Review by MatthewPyle-NOAA — COMMENTED** [review]
Had a couple of comments/questions about changes that didn't seem to fit with the rest of the PR.

File: scripts/exrrfs_run_fcst.sh
Comment (chan-hoo) [reply scripts/exrrfs_run_fcst.sh:1]: Yes. Without this argument, the `RRFS_NA_3km` run failed (this domain has another issue though). The `RRFS_CONUS_25km` run was completed successfully either with or without this argument.

Comment (chan-hoo) [reply scripts/exrrfs_run_fcst.sh:1]: If you want to remove this change in this PR, I'll do it. Please let me know.

Comment (MatthewPyle-NOAA) [reply scripts/exrrfs_run_fcst.sh:1]: No need to remove it for now - thanks for the explanation.

**Review by BenjaminBlake-NOAA — COMMENTED** [review]
The changes look good to me.  I want to complete a successful non-DA engineering test before approving.

**Comment by BenjaminBlake-NOAA** [comment]
I am going to test this PR on Cactus today.

**Review by BenjaminBlake-NOAA — APPROVED** [review]
I tested these changes with the GSL_HAFSV0.A_13km grid since that is one of the pre-defined regional_latlon grids.  The generate workflow step complained about pre-generated fix files missing even though the grid/orog/sfc_climo tasks were set to run, and the make_grid task complained about not finding a fvcom mask file.  However my non-DA engineering test otherwise completed successfully, and those errors are not related to this PR, so I approve of these changes.  Thanks @chan-hoo !

**Review by MatthewPyle-NOAA — APPROVED** [review]
Approving this one now - thanks for testing, @BenjaminBlake-NOAA

**Pull request merged**